### PR TITLE
[FrameworkBundle] PR #26213 Document redirections with 307/308 HTTP status codes

### PR DESCRIPTION
**Pull request by ZipoKing**
This documents changes introduced with pull request https://github.com/symfony/symfony/pull/26213

### CODE REVIEW

File: routing/redirect_in_config.rst [hunk 1]
```diff
@@ -155,3 +155,76 @@ action:
     Because you are redirecting to a route instead of a path, the required
     option is called ``route`` in the ``redirect()`` action, instead of ``path``
     in the ``urlRedirect()`` action.
+
+Redirecting POST/PUT calls
+--------------------------
+
+As default behaviour of both methods mentioned above results in sending
```
Comment (Simperfit) [inline routing/redirect_in_config.rst]: `As a`

File: routing/redirect_in_config.rst [hunk 2]
```diff
@@ -155,3 +155,76 @@ action:
...
+        $collection->add('admin', new Route('/webhooks/foo', array(
+            '_controller' => 'Symfony\Bundle\FrameworkBundle\Controller\RedirectController::redirectAction',
+            'route'       => 'foo_webhook_handler',
+            'permanent'   => true,
+            'keepRequestMethod' => true
+        )));
+
+        return $collection;
+
+Switching ``keepRequestMethod`` switch to ``true`` will result in sending
```
Comment (Simperfit) [inline routing/redirect_in_config.rst]: When ``keepRequestMethod`` is set to true with either ``permanent`` set to false which will lead to a ``307`` response or ``308`` with ``permanent`` being true. Theses codes will give information in the HTTP request that the method should be repeated without altering the body.

File: routing/redirect_in_config.rst [hunk 3]
```diff
@@ -155,3 +155,76 @@ action:
...
+        </routes>
+
+    .. code-block:: php
+
+        // config/routes.php
+        use Symfony\Component\Routing\RouteCollection;
+        use Symfony\Component\Routing\Route;
+
+        $collection = new RouteCollection();
+        // ...
```
Comment (Simperfit) [inline routing/redirect_in_config.rst]: you can remove this

File: routing/redirect_in_config.rst [hunk 4]
```diff
@@ -155,3 +155,76 @@ action:
...
+    .. code-block:: xml
+
+        <!-- config/routes.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <routes xmlns="http://symfony.com/schema/routing"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/routing
+                http://symfony.com/schema/routing/routing-1.0.xsd">
+
+            <!-- ... -->
```
Comment (Simperfit) [inline routing/redirect_in_config.rst]: you can remove this

File: routing/redirect_in_config.rst [hunk 5]
```diff
@@ -155,3 +155,76 @@ action:
...
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\RedirectController::redirectAction`
+are accepting aditional switch called ``keepRequestMethod``:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/routes.yaml
+
+        # ...
```
Comment (Simperfit) [inline routing/redirect_in_config.rst]: you can remove this

File: routing/redirect_in_config.rst [hunk 6]
```diff
@@ -155,3 +155,76 @@ action:
     Because you are redirecting to a route instead of a path, the required
     option is called ``route`` in the ``redirect()`` action, instead of ``path``
     in the ``urlRedirect()`` action.
+
+Redirecting POST/PUT calls
+--------------------------
+
+As default behaviour of both methods mentioned above results in sending
+response with ``301`` or ``302`` HTTP status codes, the following call will
+be made with use of HTTP request method. But it some scenarios it is
```
Comment (Simperfit) [inline routing/redirect_in_config.rst]: `use of HTTP request method`
WDYM ? 

`But it some scenarios`
But in some scenarios it's either expected or required that the following call [...]

**Comment by ZipoKing** [comment]
Thank you @Simperfit - all changes applied

**Comment by javiereguiluz** [comment]
@ZipoKing thanks for contributing the docs for this nice feature that you also contributed ... and congrats on your first Symfony Docs contribution!

**Pull request merged**